tools/bcftools_merge.cwl

```diff
@@ -0,0 +1,89 @@
+cwlVersion: v1.2
+class: CommandLineTool
+id: bcftools_merge
+doc: |
+  BCFTOOLS merge
+requirements:
+  - class: InlineJavascriptRequirement
+  - class: ShellCommandRequirement
+  - class: ResourceRequirement
+    ramMin: $(inputs.ram*1000)
+    coresMin: $(inputs.cpu)
+  - class: DockerRequirement
+    dockerPull: 'staphb/bcftools:1.19'
+  - class: InitialWorkDirRequirement
+    listing:
```

Comment:
nice

Reply:
Yup. This avoids having a crazy long command line. Very useful when you have 100s of files to merge.